### PR TITLE
Change toPersistentMap/Set/HashMap/HashSet in appliance with the discussions

### PR DESCRIPTION
**Pull request by qurbonzoda**

### CODE REVIEW

File: kotlinx-collections-immutable/tests/src/test/kotlin/kotlinx.collections.immutable/contractTests/immutableSet/ImmutableSetTest.kt [hunk 1]
```diff
@@ -24,15 +24,22 @@ import kotlin.test.*
 
 class ImmutableHashSetTest : ImmutableSetTestBase() {
     override fun <T> immutableSetOf(vararg elements: T) = persistentHashSetOf(*elements)
+    override fun <T> testBuilderToPersistentSet(builder: PersistentSet.Builder<T>) {
```
Comment (ilya-g) [inline kotlinx-collections-immutable/tests/src/test/kotlin/kotlinx.collections.immutable/contractTests/immutableSet/ImmutableSetTest.kt]: Makes sense to add a similar test for `toPersistentMap`

**Comment by ilya-g** [comment]
Also please change the commit message, it's unclear what is "the meeting" :)

File: kotlinx-collections-immutable/tests/src/test/kotlin/kotlinx.collections.immutable/contractTests/immutableSet/ImmutableSetTest.kt (hunk 1 above)
Comment (qurbonzoda) [reply kotlinx-collections-immutable/tests/src/test/kotlin/kotlinx.collections.immutable/contractTests/immutableSet/ImmutableSetTest.kt]: +1

**Comment by qurbonzoda** [comment]
> Also please change the commit message, it's unclear what is "the meeting" :)

Done

**Pull request merged**